hack/kubectl.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: kubectl
+spec:
+  volumes:
+    - name: mk
+      secret:
+        secretName: multikueue
+  containers:
+    - name: kubectl
+      image: bitnami/kubectl:latest
```

Comment:
I would prefer using the image from the k8s official repo: `registry.k8s.io/kubectl:v1.28.4`. However, maybe we could have some basic ginkgo test, which just lists nodes (or any other resource) on the worker cluster? We need to have the ginkgo tests anyway, so moving the setup to this PR seems to make sense.

Reply:
ginkgo based tests added